pkgs/by-name/ya/yazi-unwrapped/package.nix

```
@@ -25,7 +26,8 @@ rustPlatform.buildRustPackage rec {
   env.YAZI_GEN_COMPLETIONS = true;
 
   nativeBuildInputs = [ installShellFiles ];
-  buildInputs = lib.optionals stdenv.isDarwin [ Foundation ];
+  buildInputs = lib.optionals stdenv.isDarwin [Foundation]
+                ++ lib.optionals (!stdenv.isDarwin) [rust-jemalloc-sys];
```

It looks like `rust-jemalloc-sys` is also available on darwin, maybe we can make it always be used?

```suggestion
  buildInputs = [ rust-jemalloc-sys ] ++ lib.optionals stdenv.isDarwin [ Foundation ];
```

---

I have tried it on `x86_64-darwin`, and it works.

---

Thank you for trying it on darwin. I'll commit your change